src/main/java/com/example/online_shop_api/Service/PasswordService.java

```diff
@@ -7,10 +7,9 @@
 import com.example.online_shop_api.Repository.EmployeeRepository;
 import com.example.online_shop_api.Repository.UserRepository;
 import lombok.AllArgsConstructor;
-import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.Authentication;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
```

Comment:
The correct password encoder that we use is BCryptPasswordEncoder, not the one declared in the line below (12).